ui/webui/test/helpers/integration.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/python3
+#
+# Copyright (C) 2022 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import sys
+from subprocess import CalledProcessError
+
+# import Cockpit's machinery for test VMs and its browser test API
+TEST_DIR = os.path.dirname(__file__)
+sys.path.append(os.path.join(TEST_DIR, "common"))
+sys.path.append(os.path.join(TEST_DIR, "helpers"))
+sys.path.append(os.path.join(os.path.dirname(TEST_DIR), "bots/machine"))
+
+from installer import Installer
+from language import Language
+from storage import Storage
+from review import Review
+from progress import Progress
+from testlib import MachineCase  # pylint: disable=import-error
+from machine_install import VirtInstallMachine
+
+
+class IntegrationTest(MachineCase):
+    MachineCase.machine_class = VirtInstallMachine
+
+    def setUp(self):
+        super().setUp()
+        self.installer = Installer(self.browser, self.machine)
+        self.helper_language = Language(self.browser, self.machine)
```

Comment:
I don't really like the `helper_` part in the names of these attributes. Also, I recommend to keep all attributes and helper methods private to avoid collisions with `MachineCase` attributes and future test methods.

Reply:
I suggested to rename the conflicting methods in https://github.com/rhinstaller/anaconda/pull/4179#discussion_r1065527774, so these names can be taken by these attributes instead. For example, let's use `_language` to access the helper object and `_configure_language` to define the interaction with the language screen. However, maybe I am missing something.